website/components/TypeDefinition.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { css } from 'emotion';
+import Code from '@leafygreen-ui/code';
+import { InlineCode } from '@leafygreen-ui/typography';
+
+function TypeDefinition({ markdownAst, readme }) {
```

Comment:
is this file not being type-checked properly? the functions in it are missing types

Reply:
added some types but yes i think you're right